packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -363,6 +391,55 @@ export default class MeetingInfoV2 {
       });
   }
 
+  /**
+   * disables static meeting link for given conversation url
+   * @param {String} conversationUrl conversationUrl that's required to disable static meeting link if it exists
+   * @returns {Promise} returns a meeting info object
+   * @public
+   * @memberof MeetingInfo
+   */
+  async disableStaticMeetingLink(conversationUrl: string) {
+    if (!this.webex.meetings.preferredWebexSite) {
+      throw Error('No preferred webex site found');
+    }
+
+    const body = {
+      spaceUrl: conversationUrl,
+    };
+
+    const uri = this.webex.meetings.preferredWebexSite
```

Comment:
is this ternary even necessary due to the guard check
```
if (!this.webex.meetings.preferredWebexSite) {
      throw Error('No preferred webex site found');
    }
```

if there is no `this.webex.meetings.preferredWebexSite` it won't even get here anyways, right?